draft/spec/index.html

```diff
@@ -620,10 +620,10 @@ <h2>Basic Structure</h2>
                 </dd>
                 <dt><code>digestAlgorithm</code></dt>
                 <dd>
-                    The digest algorithm used for calculating digests within the OCFL Object. This
-                    <span id="E039">MUST</span> be either <code>sha512</code> or <code>sha256</code>, and
-                    <span id="W006">SHOULD</span> be <code>sha512</code>. See thead
-                    <a href="#digest-algorithms">section on Digests</a> for more information.
+                    The digest algorithm used for calculating digests for content-addressing within the OCFL Object
```

Comment:
Is content addressing really hyphenated? Seems strange

Reply:
It is 3 out of the 4 other times it appears in the spec so I'm being consistent

Reply:
It is probably one of those things that could go either way. There is something to be said for consistency... and it appears that wikipedia goes with the hyphen:
https://en.wikipedia.org/wiki/Content-addressable_storage